frontend/src/App.tsx

```diff
@@ -63,25 +60,25 @@ function MobileAppToolbar() {
   }
 
   return (
-      <Toolbar>
-        <IconButton onClick={() => setOpen(true)}>
-          <MenuIcon />
-        </IconButton>
-        <Drawer variant="persistent" anchor="left" open={open}>
-          <Button aria-label="Hide site navigation button"
-                  onClick={() => setOpen(false)}>
-            <ChevronLeftIcon />
-          </Button>
-          <List>
-            {Object.keys(PAGE_URL_TO_NAMES).map((pageUrl, index) => (
-                <ListItemLink href={pageUrl} key={index}
-                              aria-label={PAGE_URL_TO_NAMES[pageUrl]}>
-                  <ListItemText primary={PAGE_URL_TO_NAMES[pageUrl]} />
-                </ListItemLink>
-            ))}
-          </List>
-        </Drawer>
-      </Toolbar>
+    <Toolbar>
+      <IconButton onClick={() => setOpen(true)}
+                  aria-label="Expand site navigation button">
+        <MenuIcon className={styles.MenuIconForMobile} />
+      </IconButton>
+      <Drawer variant="persistent" anchor="left" open={open}>
+        <Button aria-label="Hide site navigation button"
```

Comment:
I would use "Collapse site navigation" here. "Hide" is also a fine verb, but I think expand/collapse is a more common convention. Leave out "button" so this reads like a command (the element being a <button> is enough to communicate its role).